Snowflake.Data.Tests/SFDbCommandBuilderTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests
+{
+    using NUnit.Framework;
+    using Snowflake.Data.Client;
+
+    [TestFixture]
+    class SFDbCommandBuilderTest : SFBaseTest
+    {
+        SnowflakeDbCommandBuilder builder;
+
+        [SetUp]
+        public void BeforeTest()
+        {
+            builder = new SnowflakeDbCommandBuilder();
+        }
+
+        [Test]
+        [Ignore("DbCommandBuilder")]
+        public void DbCommandBuilderDone()
+        {
+            // Do nothing;
+        }
+
+        [Test]
+        public void TestEmptyCommandBuilder()
+        {
+            Assert.AreEqual(null, builder.DataAdapter);
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_PREFIX, builder.QuotePrefix);
```

Comment:
Empty means without Data Adapter. Should we maybe move default quotations checks to a dedicated test for clarity of test purpose?

Reply:
I separated them into two tests: `TestDefaultCommandBuilder` and `TestCommandBuilderWithoutAdapter`

Is something like that what you meant?